processing/src/main/java/org/apache/druid/segment/incremental/ParseExceptionHandler.java

```diff
@@ -76,27 +86,51 @@ public void handle(@Nullable ParseException e)
       rowIngestionMeters.incrementProcessedWithError();
     } else {
       rowIngestionMeters.incrementUnparseable();
-    }
+      if (emitter != null) {
+        DateTime timeOfException = new DateTime(e.getTimeOfExceptionMillis());
 
-    logParseExceptionHelper(e);
-
-    if (savedParseExceptionReports != null) {
-      ParseExceptionReport parseExceptionReport = new ParseExceptionReport(
-          e.getInput(),
-          e.isFromPartiallyValidRow() ? "processedWithError" : "unparseable",
-          e.isFromPartiallyValidRow()
-          ? ((UnparseableColumnsParseException) e).getColumnExceptionMessages()
-          : ImmutableList.of(e.getMessage()),
-          e.getTimeOfExceptionMillis()
-      );
-      savedParseExceptionReports.add(parseExceptionReport);
-    }
+        ImmutableMap<String, Object> extraData = new ImmutableMap.Builder().putAll(taskMetadata)
+                                                                           .put("input", e.getInput())
+                                                                           .put("detailMessage", e.getMessage())
+                                                                           .build();
+        Event event = new AlertEvent(
+            timeOfException,
+            "peon",
```

Comment:
Should service and host be populated by emitter at deeper level? `localhost` wont always be the case here.

Reply:
In elsewhere in the code, it seems like `ServiceEmitter` would get constructed with the service/host in `EmitterModule`, and:

```

  public void emit(ServiceEventBuilder builder)
  {
    emit(builder.build(serviceDimensions));
  }
```

would be called to attach those dimensions to the event

Reply:
Yeah, in addition to the configurable host property, I think the service can also be an "indexer" if configured for task execution.